apps/condo/domains/banking/access/CreateBankAccountRequestService.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * Generated by `createservice banking.CreateBankAccountRequestService '--type=mutations'`
+ */
+const get = require('lodash/get')
+const { throwAuthenticationError } = require('@open-condo/keystone/apolloErrorFormatter')
+const { checkOrganizationPermission } = require('@condo/domains/organization/utils/accessSchema')
+
+async function canCreateBankAccountRequest ({ authentication: { item: user }, args: { data } }) {
+    if (!user) return throwAuthenticationError()
+    if (user.deletedAt) return false
+    if (user.isAdmin || user.isSupport) return true
+
+    const organizationId = get(data, 'organizationId')
+    if (!organizationId) return false
+
+    return await checkOrganizationPermission(user.id, organizationId, 'canManageProperties')
```

Comment:
It's a bit confusing. Why properties access is used in banking?

Reply:
use canManageBankAccount